constellations/gemini-admin/src/home/home.js

```diff
@@ -1,108 +1,110 @@
-                            eruditi copiosae contentiones cu. Sed ea quando
-                            mediocrem, soleat dolorum no ius. Eu electram
-                            iracundia mnesarchum his, eum ut hinc latine. Est
-                            simul definiebas ut, debitis invenire eu usu. Eu
-                            eros appetere mel, nullam delenit tincidunt duo at.
-                            Eos option appetere torquatos in, cu debitis
-                            singulis principes usu.
-                        </p>
-                    </div>
-                </div>
+                <div className="showSection">{sectionComponent}</div>
+            </div>
+        );
+    }
+}
 
-                <div className="section">
-                    <h3 className="section-header">New Registrations</h3>
-                    <div className="container-registration">
-                        <p>
-                            Lorem ipsum dolor sit amet, vim ei tota dicant
-                            interpretaris, sea consulatu scripserit ei, pri eu
-                            accumsan contentiones. Usu ei iriure deleniti. Cum
-                            ut periculis laboramus referrentur. Vis ea laoreet
-                            imperdiet deterruisset, probo pertinax iudicabit qui
-                            ea. Duo liber quodsi contentiones ne, vel ei movet
-                            conceptam, id his quod iriure feugiat. Qui ex vide
-                            labitur volumus. His cu fugit adolescens
-                            voluptatibus, et per torquatos interesset. Iusto
-                            deleniti invenire id cum. Te saepe alterum
-                            appellantur pro, in est errem dicant suscipit,
-                            oblique argumentum sed et. Quo cu partem inermis,
-                            vix et minimum vivendum, ut illud delectus eos. Mei
-                            te alia justo, per amet nemore quodsi cu, mel
-                            eruditi copiosae contentiones cu. Sed ea quando
-                            mediocrem, soleat dolorum no ius. Eu electram
-                            iracundia mnesarchum his, eum ut hinc latine. Est
-                            simul definiebas ut, debitis invenire eu usu. Eu
-                            eros appetere mel, nullam delenit tincidunt duo at.
-                            Eos option appetere torquatos in, cu debitis
-                            singulis principes usu.
-                        </p>
-                    </div>
-                </div>
+class TabButton extends React.Component {
+    state = {
+        isZero: false,
+    };
+
+    render() {
+        let highlight = this.props.highlight;
+        let section = this.props.section;
+        let displayName = sectionDisplayNames[section];
+        let numNotif = this.props.buttonNum;
+
+        if (numNotif == 0) {
+            this.setState({
```

Comment:
having trouble here. state variable isZero isn't updating so "zero" can't be added to the className for displaNotif. need to add "zero" for css